source/deprecations/v2.x.html.md

```diff
@@ -609,9 +609,48 @@ After:
 ```js
 import Ember from 'ember';
 const { computed } = Ember;
+
 export default Ember.Component.extend({
   myString: computed(function(){
     return Ember.String.htmlSafe(someString);
   });
 )};
 ```
+
+Detecting safe strings. Before:
+
+```js
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  actions: {
+    save() {
+      let myString = this.get('myString');
+      if (myString instanceof Ember.Handlebars.SafeString) {
+        // ...
+      }
+    }
+  }
+});
+```
+
+After:
+
+```js
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  actions: {
+    save() {
+      let myString = this.get('myString');
+      if (Ember.String.isHtmlSafe(myString)) {
+        // ...
+      }
+    }
+  }
+});
+```
+
+If you're an addon maintainer, there is a polyfill for safe string detection ([ember-string-ishtmlsafe-polyfill](https://github.com/workmanw/ember-string-ishtmlsafe-polyfill))
+that will help maintain backwards compatibility. Additionally, it's worth noting that `Ember.Handlebars.htmlSafe`
```

Comment:
I believe this should be `Ember.String.htmlSafe` (not `Ember.Handlebars.htmlSafe`).